src/sfProject.ts

```diff
@@ -31,30 +36,7 @@ export type PackageDirDependency = {
   versionNumber?: string;
 };
 
-export type PackageDir = {
-  ancestorId?: string;
-  ancestorVersion?: string;
-  default?: boolean;
-  definitionFile?: string;
-  dependencies?: PackageDirDependency[];
-  includeProfileUserLicenses?: boolean;
-  package?: string;
-  packageMetadataAccess?: {
-    permissionSets: string | string[];
-    permissionSetLicenses: string | string[];
-  };
-  path: string;
-  postInstallScript?: string;
-  postInstallUrl?: string;
-  releaseNotesUrl?: string;
-  scopeProfiles?: boolean;
-  uninstallScript?: string;
-  versionDescription?: string;
-  versionName?: string;
-  versionNumber?: string;
-  unpackagedMetadata?: { path: string };
-  seedMetadata?: { path: string };
-};
+export type PackageDir = PackageDirFromSchemas;
```

Comment:
maybe add a comment to remove in the next major, and have dependencies rely on the `/schemas` instead?